### PR TITLE
feat(modbus): add testing tool for modbus go tests

### PR DESCRIPTION
**Pull request by gq0**
In order to run the Modbus lib test suite we need some more tools, including this rather old freeware:
http://www.modbusdriver.com/diagslave.html

I wasn't able to upload a new image, I guess I don't have the right to do it.

### CODE REVIEW

**Comment by jhedev** [comment]
I think it would be better to put this into another image based on the golang-tools image.

**Comment by gq0** [comment]
Can be done.

**Comment by gq0** [comment]
Still testing it, the internet is soooo slow here.

File: modbus-tools/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ARG GO_VERSION=
+FROM golang-tools:${GO_VERSION}
```
Comment (jhedev) [inline modbus-tools/Dockerfile]: This should be gridx/golang-tools

**Review by jhedev — CHANGES_REQUESTED** [review]

**Comment by jhedev** [comment]
as soon as we have merged I can rebuild and push it

File: modbus-tools/Dockerfile (hunk 1 above)
Comment (gq0) [reply modbus-tools/Dockerfile]: changed to pure golang image

Comment (gq0) [reply modbus-tools/Dockerfile]: Oh, shit ne, it needs glide.

**Review by jhedev — APPROVED** [review]

**Pull request merged**